pkg/routeagent_driver/constants/constants.go

```diff
@@ -38,6 +38,9 @@ const (
 	// [#] interface on the node that has an IPAddress from the clusterCIDR
 	CniInterfaceIP = "submariner.io/cniIfaceIp"
 
+	// TODO (revisit): Unify this constant across different components
+	SubmarinerIpamGlobalIp = "submariner.io/globalIp"
```

Comment:
`Submariner` -> `Sm` for consistency?

```suggestion
	SmGlobalIP = "submariner.io/globalIp"
```